code/modules/mob/living/simple_animal/bees/bees_datums.dm

```diff
@@ -55,6 +55,7 @@ var/bees_count = 0
 
 /datum/bee/proc/death(var/gibbed = FALSE)
 	if (mob)
+		mob.bees.Remove(src)
```

Comment:
This already happens in `Destroy()`, so this won't do anything

Reply:
gah

Reply:
I will keep looking around